File: abc/abc139/abc139_a.cs
// https://atcoder.jp/contests/abc139/submissions/7252682

using System;

class Program
{
    static void Main()
    {
        var S = Console.ReadLine();
        var T = Console.ReadLine();

        var counter = 0;
        for (int i = 0; i < S.Length; i++)
        {
            if (S[i] == T[i])
            {
                counter++;
            }
        }
        Console.WriteLine(counter);
    }
}

File: abc/abc139/abc139_a.fs
// https://atcoder.jp/contests/abc139/submissions/12017644

let S = stdin.ReadLine()
let T = stdin.ReadLine()

[ 0..2 ]
|> List.map (fun x ->
    if S.[x] = T.[x] then 1
    else 0)
|> List.sum
|> stdout.WriteLine
